benchmarking/utils.py

```diff
@@ -1,5 +1,10 @@
+from typing import List
+
 def parse_multiple_id_regex(ids):
     """Parses ids [x,y,x] to the form "/^(x|y|z)$/"""
     string_ids = [str(id) for id in ids]
     pipe_separated_ids = "|".join(string_ids)
     return f"/^({pipe_separated_ids})$/"
+
+def get_revision_intervals(timestamps:list) -> List[tuple]:
```

Comment:
```suggestion
def get_revision_intervals(timestamps:List[str]) -> List[tuple]:
```
Vil anta det var dette du ville?

Reply:
Suggestions er helt vilt ass

Reply:
Oops, ja! Thxthx